docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,195 @@
+---
+title: DateTime and DateTimeOffset support in System.Text.Json
+description: An overview of how DateTime and DateTimeOffset types are supported in the System.Text.Json library.
+ms.technology: dotnet-standard
+author: layomia
+ms.author: laakinri
+ms.date: 07/22/2019
+helpviewer_keywords:
+  - "JSON, Serializer, Utf8"
+  - "JSON DateTime, JSON DateTimeOffset"
+  - "DateTime, DateTimeOffset"
+  - "JsonSerializer, Utf8JsonReader, Utf8JsonWriter, JsonElement, JsonDocument"
+  - "JSON Serializer, JSON Reader, JSON Writer"
+  - "Converter, JSON Converter, DateTime Converter"
+  - "ISO, ISO 8601, ISO 8601-1:2019"
+---
+# DateTime and DateTimeOffset support in System.Text.Json
+
+The System.Text.Json library offers flexible support for formatting and parsing both <xref:System.DateTime> and <xref:System.DateTimeOffset> values.
```

Comment:
I don't think that statement is true. In fact, we explicitly only support ISO format (so not super flexible), particularly for formatting/parsing. There is **customization** support for flexible **serializing/deserializing** though.